### PR TITLE
Added stochastic plots for SAIR and SEIR

### PR DESCRIPTION
**Pull request by rizfn**

### CODE REVIEW

File: source/epidemiology-other-compartmental-models.rst [hunk 1]
```diff
@@ -56,6 +56,17 @@ If the agent is already infected, we transition them to the ``Removed`` compartm
...
 
+Similarly to the SIR model, the average of the stochastic solutions should represent the mean field solution, as seen in the image below
+
+.. figure:: _static/images/epidemiology_SEIR_stochastic.png
+  :align: center
+  :alt: The stochastic solutions to the differential equations of the SEIR Model
+  :width: 600px
+  :figclass: align-center
+
+  $\lambda_S$=0.6/week, $\lambda_E$=0.5/week, $\lambda_I$=0.25/week
```
Comment (dpcherian) [inline source/epidemiology-other-compartmental-models.rst:67]: Rates should be per day, not per week.

File: source/epidemiology-other-compartmental-models.rst [hunk 2]
```diff
@@ -56,6 +56,17 @@ If the agent is already infected, we transition them to the ``Removed`` compartm
 
  P_\text{IR} = \lambda_I \Delta t.
 
+Similarly to the SIR model, the average of the stochastic solutions should represent the mean field solution, as seen in the image below
+
+.. figure:: _static/images/epidemiology_SEIR_stochastic.png
```
Comment (dpcherian) [inline source/epidemiology-other-compartmental-models.rst:61]: Please make the `y` axis the _fraction_ of the total population.

File: source/epidemiology-other-compartmental-models.rst (hunk 1 above)
Comment (dpcherian) [inline source/epidemiology-other-compartmental-models.rst:67]: Mention the initial conditions as well, how is the infection seeded?

**Review by dpcherian — CHANGES_REQUESTED** [review]
Small changes, let me know when they're done.

File: source/epidemiology-other-compartmental-models.rst (hunk 1 above)
Comment (rizfn) [reply source/epidemiology-other-compartmental-models.rst:67]: I initially chose figures that'd be pretty simple in decimals per week. `lambda_E`, for instance, is 1/14 (once every two weeks). If you want me to express it per day, should I use a fraction (1/14 per day) over a decimal (~0.0714)?

File: source/epidemiology-other-compartmental-models.rst [hunk 3]
```diff
@@ -133,7 +135,9 @@ Once again, we can see the differential equation solutions as the average of the
   :width: 600px
   :figclass: align-center
 
-  $\lambda_S$=0.6/week, $\lambda_A$=0.25/week, $\lambda_I$=0.25/week, $\gamma$=0.6
+  $\lambda_S$=1.2/14 per day, $\lambda_A$=1/28 per day, $\lambda_I$=1/28 per day, $\gamma$=0.6, $N$=10,000
```
Comment (dpcherian) [inline source/epidemiology-other-compartmental-models.rst:138]: `1.2/14` looks clunky. In general, try to either get it as decimals (0.4, 0.3, 0.08, etc.) or as more or less simple fractions (1/4, 1/7, 1/10, 1/14, etc.). Also, perhaps `(day)^{-1}` would be a better unit. Fine for now, though, I'm merging this.

**Review by dpcherian — APPROVED** [review]

**Pull request merged**